Kickstarter-iOS/DataSources/HelpDataSource.swift

```diff
@@ -4,7 +4,7 @@ import KsApi
 final class HelpDataSource: ValueCellDataSource {
   func configureRows() {
     _ = HelpSectionType.allCases.map { section -> Void in
-      let values = section.cellRowsForSection.map { SettingsCellValue(user: nil, cellType: $0) }
```

Comment:
Made this optional so that we don't have to explicitly pass `nil`s all over the place